device/CMakeLists.txt

```diff
@@ -38,12 +37,12 @@ target_link_libraries(
     PUBLIC
         nng
         flatbuffers
+        uv
```

Comment:
I thought I had already made this public

Reply:
This PR has 2 "bonus" commits.  Only the last one is my change.  If the 2 former commits have been merged to `main` and cherry-pick'd back to this branch, then this branch should merge to main so that Git sees in its commit graph that those commits are part of shared history.

Reply:
Actually that probably won't work if UMD follows the same rewrite-all-commit-IDs-on-merge that tt-metal does 🥴

Reply:
Yes, please rebase to the latest main without the first two commits

Reply:
Done.